features/sriov/03-sriov-subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subsription
+  namespace: sriov-network-operator
+spec:
+  channel: "4.2"
```

Comment:
If you make it parametrizable, we don't need to change it per version:

channel: "${ocp_version}"

Reply:
Thanks! I realized I missed some parameters. I've pushed some changes.